vcpkg/get_started/get-started-vs.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Install and use packages with vcpkg in Visual Studio
+description: Tutorial guides the user through the process of installing and using packages with vcpkg and Visual Studio.
+zone_pivot_group_filename: zone-pivot-groups.json
+zone_pivot_groups: operating-systems
+author: JavierMatosD
+ms.author: javiermat
+ms.topic: tutorial
+ms.date: 09/27/2023
+ms.prod: vcpkg
+#CustomerIntent: As a beginner C++ developer, I want to learn how to install and manage packages using vcpkg, so that I can easily set up and maintain C++ projects with necessary dependencies.
+---
+
+# Tutorial: Install and use packages with vcpkg
+
+This tutorial shows you how to create a C++ "Hello World" program that uses the `fmt` library with CMake, vcpkg and Visual Studio. You'll install dependencies, configure, build, and run a simple application.
+
+## Prerequisites
+
+- [Visual Studio with C++ development workload](https://visualstudio.microsoft.com/downloads/)
+- [CMake](https://cmake.org/download/)
```

Comment:
```suggestion
```

VS comes with CMake. We should probably note that the user needs to install the CMake recommended component.